benchmark.py

```
@@ -0,0 +1,833 @@
+
+import aiohttp
+import asyncio
+import time
+import random
+import argparse
+import msgpack
```

Ruff (0.12.2)

`msgpack` imported but unused

Remove unused import: `msgpack`

(F401)